app/assets/stylesheets/projects.scss

```diff
@@ -735,11 +735,17 @@ li.module-hover {
         border-radius: 4px;
         box-shadow: $flyout-shadow;
 
+        .header-row {
+          display: flex;
+          width: 100%;
+        }
+
         .project-name-cell {
           align-items: center;
           display: flex;
+          flex-grow: 1;
           height: 2em;
-          margin: 0 .5em 0 2.5em;
+          margin: 0 .5em 0 1.7em;
```

Comment:
Remove flex-grow, and round to 2em

Reply:
1.7em it's more pleasing to the eyes and more like on mockups.